modules/display.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-mutable-exports */
+import Book from './book.js';
+import { DateTime } from './luxon.js';
+import {
+  listSection,
+  newBookSection,
+  contactSection,
+  bookTitle,
+  author,
+  error,
+  displayBookList,
+} from './variables.js';
+
+const books = new Book();
+let bookListArray = books.getBook();
+
+const displayBooks = () => {
+  displayBookList.innerHTML = '';
+  bookListArray.forEach((book) => displayBookList.insertAdjacentHTML(
+    'beforeend',
```

Comment:
[OPTIONAL] Good  job implementing the changes requested by the previous code reviewer but I noticed you disabled eslint in this file, it is against microverse policy to disable linter because linters help us right clean code so it would be nice if you please enable eslint . Please do this in all files were eslint has been disabled